src/components/ui/hook-form/timeslots/index.tsx

```diff
@@ -89,7 +96,11 @@ const TimeSlotsBase: FC<TimeSlotsBaseProps> = ({
                                                         onChange(nextValue);
                                                     }}
                                                 >
-                                                    {slot.label}
+                                                    {locale === 'en'
+                                                        ? convertToAmPm(
+                                                              slot.label,
```

Comment:
When we convert to am and pm can we remove :00 from labels?

Reply:
<img width="513" alt="image" src="https://github.com/ledenev-viktor/music-studio/assets/48691234/be390ae9-8ed9-42ef-9757-ab2aff6c5053">